src/http2_connection.erl

```diff
@@ -514,9 +514,10 @@ route_frame(F={H=#frame_header{
             %% Make window size great again
             lager:info("[~p] Stream ~p WindowUpdate ~p",
                        [Conn#connection.type, StreamId, L]),
-            send_window_update(self(), L),
-            gen_fsm:send_all_state_event(StreamPid, {send_window_update, L});
-        _ ->
+
+            gen_fsm:send_all_state_event(StreamPid, {send_window_update, L}),
+            send_window_update(self(), L);
+        _Tried ->
```

Comment:
nope. I was debugging something where the `send_window_update(self(), L)` wasn't sending, so one of the things I tried was moving it. It was something else, but it didn't matter here so I didn't bother changing it back